easybuild/easyconfigs/g/GObject-Introspection/GObject-Introspection-1.54.1-intel-2018a-Python-2.7.14.eb

```diff
@@ -0,0 +1,49 @@
+easyblock = 'ConfigureMake'
+
+name = 'GObject-Introspection'
+version = '1.54.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://wiki.gnome.org/GObjectIntrospection/'
+description = """GObject introspection is a middleware layer between C libraries
+ (using GObject) and language bindings. The C library can be scanned at
+ compile time and generate a metadata file, in addition to the actual
+ native C library. Then at runtime, language bindings can read this
+ metadata and automatically provide bindings to call into the C library."""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+source_urls = [FTPGNOME_SOURCE]
+sources = [SOURCELOWER_TAR_XZ]
+checksums = ['b88ded5e5f064ab58a93aadecd6d58db2ec9d970648534c63807d4f9a7bb877e']
+
+dependencies = [
+    ('Python', '2.7.14'),
+    ('GLib', '2.54.3'),
+    ('libffi', '3.2.1'),
+]
+
+builddependencies = [
+    ('Autotools', '20170619'),
+    ('flex', '2.6.4'),
+    ('Bison', '3.0.4'),
+    ('cairo', '1.14.12'),
+]
```

Comment:
It seems that `pkg-config` is required as `builddependency`:

```
checking for pkg-config... no
checking for flex... flex
checking lex output file root... lex.yy
checking lex library... -lfl
checking whether yytext is a pointer... yes
checking for bison... bison -y
checking for dlopen in -ldl... yes
checking for the suffix of shared libraries... .so
checking for GLIB... no
configure: error: in `/mnt/ramdisk/GObjectIntrospection/1.54.1/intel-2018a-Python-2.7.14/gobject-introspection-1.54.1':
configure: error: The pkg-config script could not be found or is too old.  Make sure it
is in your PATH or set the PKG_CONFIG environment variable to the full
path to pkg-config.

Alternatively, you may set the environment variables GLIB_CFLAGS
and GLIB_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details.

To get pkg-config, see <http://pkg-config.freedesktop.org/>.
```

Reply:
fixed in 92fd34d + #6216